src/drops.cpp

```diff
@@ -103,7 +103,7 @@ drops::generate_return_value drops::do_unbind(name from, name to, asset quantity
    // Calculate amount of RAM needing to be purchased
    // NOTE: Additional RAM is being purchased to account for the buyrambytes bug
    // SEE: https://github.com/EOSIO/eosio.system/issues/30
-   uint64_t ram_purchase_amount = unbinds_itr->drops_ids.size() * (record_size + purchase_buffer);
+   int64_t ram_purchase_amount = get_bytes_per_drop();
```

Comment:
This should still be multiplying by the number of drops, yeah?

```
int64_t ram_purchase_amount = unbinds_itr->drops_ids.size() * get_bytes_per_drop();
```

Reply:
Ah yes! you are right

I'm thinking of adding an `amount` param to `get_bytes_per_drop(int amount)`